ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -94,7 +94,13 @@ cat > packet-config.yaml <<-EOF
 EOF
 ansible-playbook packet-config.yaml |& gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0; fflush(); }'
 
+#if [ -e "${CLUSTER_PROFILE_DIR}/ofcir_url" ] && [ "${ARCHITECTURE}" != "arm64" ] ; then
```

Comment:
^

Reply:
Yes we should remove the comments.  Will resubmit shortly.